tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -289,47 +290,96 @@ def pyramid_request(self, pyramid_request):
         return pyramid_request
 
 
+class TestConfigureModuleItem(ConfiguredLaunch):
+    def make_request(self, context, pyramid_request):
+        BasicLTILaunchViews(context, pyramid_request).configure_module_item()
```

Comment:
The real diff here is the addition of the `ConfiguredLaunch` base class. The rest is a result of moving it to be above the un-configured launches.

I've also used the `make_request()` function as in other classes to highlight the similarity between them.

Reply:
I've moved this and the others down. I didn't do this in the parent class as it's not really running tests, and you need to know front and center that you are expected to implement that function.

Reply:
:+1: